source/leuven/genius_quick_start.rst

```diff
@@ -71,10 +71,35 @@ walltimes of 3 days or less.
 
 Submit to a compute node
 ~~~~~~~~~~~~~~~~~~~~~~~~
-To submit to a compute node it all boils down to specifying the required number of nodes and cores. As the nodes have a single user policy we recommend to always request all available cores per node (36 cores in this case). For example to request 2 nodes with each 36 cores you can submit like this::
+Submitting a compute job boils down to specifying the required number of nodes, cores-per-node, memory and walltime.
+You may e.g. request two full nodes like this:
 
    $ qsub -l nodes=2:ppn=36  -l walltime=2:00:00  -A myproject  myjobscript.pbs
-  
+
+However, you may request a portion of resources on a node.
+For instance, to test a multi-threaded application which performs optimally using 4 cores, you may submit your job like this:
+
+   $ qsub -l nodes=1:ppn=4  -l walltime=2:00:00  -A myproject  myjobscript.pbs
+   
+In the two above examples, the jobs may start on Skylake or Cascadelake nodes.
+Please bear in mind to not exceed the maximum allowed resources on compute nodes for the targeted partition.
+E.g. you can request at most 36 cores per node (``ppn=36``).
+
+Advanced node usage
+^^^^^^^^^^^^^^^^^^^
+The node access policy on Skylake nodes is `SINGLEUSER`.
+This means that once a job lands on a Skylake node(s), no job from other users can land on the same node(s).
+If you insist on using a full node (to exclude jobs from other users), you may enforce getting a Skylake node in one of the following ways
+
+   $ qsub -l nodes=1:ppn=8:skylake -l walltime=30:00 -A myproject myjobscript.pbs             # or
+   $ qsub -l nodes=1:ppn=8 -l feature=skylake -l walltime=30:00 -A myproject myjobscript.pbs
+
+The node access policy on Cascadelake nodes is `SHARED`.
+This means the CPU and memory resources per nodes are exploited as much as possible by packing more and more jobs into a single node.
+Similarly, you may enforce getting a Cascadelake node by specifying either `nodes=1:ppn=8:cascadelake` or `-l feature=cascadelake`
```

Comment:
I think the formatting here is nicer when you but double back quotes around the resource specifications, so \`\`nodes=1:ppn=8:cascadelake\`\` and  \`\`-l feature=cascadelake\`\`

Reply:
Thanks @stevenvdb for the formatting suggestions. I fixed them all.